packages/react-aria-components/docs/Tree.mdx

```diff
@@ -322,6 +833,108 @@ TreeItem also exposes a `--tree-item-level` CSS custom property, which you can u
 }
 ```
 
+### TreeItemContent
+
+A `TreeItemContent` can be targeted with the `.react-aria-TreeItemContent` CSS selector, or by overriding with a custom `className`. It supports the following states and render props:
+
+<StateTable properties={docs.exports.TreeItemContentRenderProps.properties} showOptional />
```

Comment:
Missing a bunch of the data attributes

Reply:
hmm since TreeItemContent doesn't actually render a DOM node we should just remove this section entirely